### ISSUE TITLE
Latest Nest CLI not compatible with latest prettier 

### ISSUE THREAD

**Issue by elio-maggini** [nestjs/nest-cli#1065]
<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When running lint from command line latest release of prettier (8.1.0) throws an error that "prettier/@typescript-eslint" had been merged into prettier.

The issue is located within node_modules/@nestjs/cli/.eslintrc.js 

```
  extends: [
    'plugin:@typescript-eslint/eslint-recommended',
    'plugin:@typescript-eslint/recommended',
    'prettier',
    'prettier/@typescript-eslint',
  ],
  ```

## Expected behavior

Nest CLI should run properly.

## Minimal reproduction of the problem with instructions

Update prettier and NestJS CLI

## What is the motivation / use case for changing the behavior?

The libraries dependencies should remain up-to-date

## Environment

```


[System Information]
OS Version     : Windows 10
NodeJS Version : v12.19.0
YARN Version    : 1.22.5

[Nest CLI]
Nest CLI Version : 7.5.6

[Nest Platform Information]
platform-express version : 7.6.13
serve-static version     : 2.1.4
graphql version          : 7.4.1
common version           : 7.6.13
core version             : 7.6.13
```

**Comment by kamilmysliwiec** [comment]
Fixed in `@nestjs/schematics@7.2.8`. Thanks for reporting!

**Comment by elio-maggini** [comment]
Hello

Using the latest and greatest I am still setting this issue during linting.

```
[System Information]
OS Version     : Windows 10
NodeJS Version : v12.19.0
YARN Version    : 1.22.5

[Nest CLI]
Nest CLI Version : 7.5.6

[Nest Platform Information]
platform-express version : 7.6.13
serve-static version     : 2.1.4
graphql version          : 7.4.1
common version           : 7.6.13
core version             : 7.6.13
```

```
> nx run XXX:lint 

Linting "XXX"...
Cannot read config file: C:\Users\MaggiEl\dev\XXX\node_modules\eslint-config-prettier\@typescript-eslint.js
Error: "prettier/@typescript-eslint" has been merged into "prettier" in eslint-config-prettier 8.0.0. See: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#ver
sion-800-2021-02-21
Referenced from: C:\Users\MaggiEl\dev\XXX\node_modules\@nrwl\eslint-plugin-nx\src\index.js
```
